shared/src/test/diff/qq/NuSyntax.mls

```diff
@@ -158,7 +158,7 @@ fun foo(x) =
 `(`42)
 //│ ╔══[ERROR] Nested quotation is not allowed.
 //│ ║  l.158: 	`(`42)
-//│ ╙──       	   ^^
+//│ ╙──       	  ^^^
```

Comment:
I wonder why this location is pointing at blank space...

Reply:
Isn't the space inside a diff message? 🤔

Reply:
Oh never mind, it was just a rendering bug when looking at this from my phone!